samples/model-builder/create_training_pipeline_image_classification_sample.py

```diff
@@ -33,7 +34,10 @@ def create_training_pipeline_image_classification_sample(
 ):
     aiplatform.init(project=project, location=location)
 
-    job = aiplatform.AutoMLImageTrainingJob(display_name=display_name)
+    job = aiplatform.AutoMLImageTrainingJob(display_name=display_name,
+                                            prediction_type='classification',
+                                            multi_label=multi_label
+                                           )
```

Comment:
nit: re-indent them (lint might make you change this anyway) like the code below on line 44:

```
model = job.run(
        dataset=my_image_ds,
        model_display_name=model_display_name,
        training_fraction_split=training_fraction_split,
        validation_fraction_split=validation_fraction_split,
        test_fraction_split=test_fraction_split,
        budget_milli_node_hours=budget_milli_node_hours,
        disable_early_stopping=disable_early_stopping,
        sync=sync,
    )
```

Reply:
fixed indentation